Code/ecg.py

```diff
@@ -0,0 +1,35 @@
+class ECG:
+
+    def __init__(self, file, window, brady_max, tachy_min):
```

Comment:
Need a docstring for `__init__()`.

Reply:
Added docstrings to all functions in ECG class